internals/notifier_test.py

```diff
@@ -469,9 +470,14 @@ def test_get_feature_starrers__some_starrers(self):
     notifier.FeatureStar.set_star(app_user_1.email, feature_1_id)
     notifier.FeatureStar.set_star(app_user_2.email, feature_1_id)
 
+
+    user_1_email = app_user_1.email
+    user_2_email = app_user_2.email
+    app_user_1.key.delete()
+    app_user_2.key.delete()
     actual = notifier.FeatureStar.get_feature_starrers(feature_1_id)
     self.assertCountEqual(
-        [app_user_1.email, app_user_2.email],
+        [user_1_email, user_2_email],
```

Comment:
This is another change to remove users that persisted and caused issues in other tests.